package.json

```
@@ -37,8 +37,8 @@
         "resolve": "^1.22.1"
     },
     "peerDependencies": {
-        "@lwc/module-resolver": "^2.0.0",
-        "lwc": "^2.0.0"
+        "@lwc/module-resolver": "^3.0.0",
+        "lwc": "^3.0.0"
```

```suggestion
        "@lwc/module-resolver": ">= 2",
        "lwc": ">= 2"
```

`lwc-webpack-plugin` is compatible with v2 and v3. Marking it as `^3` would technically be a breaking change.